tests/foreman/cli/test_repository.py

```diff
@@ -343,47 +368,43 @@ def test_positive_create_9(self):
         content_type = u'docker'
         new_repo = self._make_repository({
             u'name': u'busybox',
+            u'docker-upstream-name': u'busybox',
             u'url': DOCKER_REGISTRY_HUB,
             u'content-type': content_type,
         })
         # Assert that urls and content types matches data passed
-        self.assertEqual(
-            new_repo['url'],
-            DOCKER_REGISTRY_HUB,
-            "Expected URL {0} but received {1}".format(
-                DOCKER_REGISTRY_HUB, new_repo['url'])
-        )
-        self.assertEqual(
-            new_repo['content-type'],
-            content_type,
-            "Expected content type {0} but received {1}".format(
-                content_type, new_repo['content-type'])
-        )
+        self.assertEqual(new_repo['url'], DOCKER_REGISTRY_HUB)
```

Comment:
:+1: This code is easier to read and less fragile, in addition to what @elyezer said.